src/Microsoft.AspNetCore.Blazor.Browser.JS/src/Boot.Server.ts

```diff
@@ -0,0 +1,39 @@
+import '../../Microsoft.JSInterop/JavaScriptRuntime/src/Microsoft.JSInterop';
+import './GlobalExports';
+import * as Environment from './Environment';
+import * as signalR from '@aspnet/signalr';
+import { MessagePackHubProtocol } from '@aspnet/signalr-protocol-msgpack';
+import { OutOfProcessRenderBatch } from './Rendering/RenderBatch/OutOfProcessRenderBatch';
+import { internalFunctions as uriHelperFunctions } from './Services/UriHelper';
+import { renderBatch } from './Rendering/Renderer';
+
+function boot() {
+  const connection = new signalR.HubConnectionBuilder()
+    .withUrl('/_blazor')
```

Comment:
I don't think I verified this works with non-root URLs (nonempty `PathBase`) so it's possible we might have to tweak something here.